bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/Page.java

```diff
@@ -369,4 +370,15 @@ default String getExportedType() {
     default String getMainContentSelector() {
         return null;
     }
+
+    /**
+     * Returns a list of items that should be included in the page
```

Comment:
I'd specifiy that there are HTML items:
Returns a list of HTML items that should be included in the page

Reply:
Done in 9973de3